include/RE/H/HUDNotifications.h

```diff
@@ -12,29 +15,24 @@ namespace RE
 	public:
 		inline static constexpr auto RTTI = RTTI_HUDNotifications;
 
-		struct UnkData1
+		class Notification
 		{
-			struct UnkData2
-			{
-				void*         unk00;  // 00
-				std::uint64_t unk08;  // 08
-				void*         unk10;  // 10
-				std::uint64_t unk18;  // 18
-			};
-			static_assert(sizeof(UnkData2) == 0x20);
+		public:
+			Notification() :
+				type(0), quest(nullptr), word(nullptr), time(0) {}
```

Comment:
Please use default member initializers instead of defining a ctor.

Reply:
@Ryan-rsm-McKenzie if I remember correctly, passing the `Notification` object that wasn't initialized to `0/nullptr/nullptr/0` to some related internal functions used to crash the game. Are you sure you want this change?

Reply:
I suggest you read up on [default member initialization](https://en.cppreference.com/w/cpp/language/data_members#Member_initialization)